packages/cli/test/integration/publicApi/workflows.test.ts

```diff
@@ -204,7 +207,7 @@ test('GET /workflows should return all owned workflows filtered by tag', async (
 		version: 1,
 	});
 
-	const tag = await testDb.createTag({});
+	const tag = await testDb.createTag({ name: 'tag', id: 1 });
```

Comment:
I tried running the tests without this addition and it seemed to work fine - the internal function generates a random name and the ID is assigned by the database.

What errors have you faced with the previous implementation?

Reply:
Yea that's true, but would name and id be undefined?

Reply:
Not really, the name would be a randomly generated one and the ID is assigned by the db